### ISSUE TITLE
Change Documentation

### ISSUE THREAD

**Issue by Seabizkit** [dotnet/ef6#1508]
With the recent release style it becoming really hard to actually know what was fix/changed in each release.

Could someone en-lighten me as to how to understand the new format, as it will mostly take you to links about dotnet core in general apposed to what was actually done to the specific repo.

example go to: https://github.com/aspnet/EntityFramework6/releases

for release .NET Core 3.1.0-preview3 baring in mind i am under EntityFramework6 and not core, it
shows a link for "Release Notes" great..  but his take you to a link about 
".NET Core 3.1.0 Preview 3 - November 14, 2019"

also for release v6.4.0, no links, you can click on and it will show 
"167 commits to master since this tag"
from this i can not even find what the actual changes were in a readable way... mostly automated commits.

I preferred the much older way of doing this where there was actually just a summary of the commits made in a readable format.

thoughts?...

**Comment by ajcvickers** [comment]
@Seabizkit Agreed that the standard way the releases are created is not very helpful. I have created a [release for 6.4](https://github.com/aspnet/EntityFramework6/releases/tag/v6.4.0) with a link to the issues fixed.

**Comment by StingyJack** [comment]
@ajcvickers - that link 404's, btw
![image](https://user-images.githubusercontent.com/15381181/83359057-b43a1e00-a345-11ea-8dd8-07d36585ea88.png)
![image](https://user-images.githubusercontent.com/15381181/83359065-c320d080-a345-11ea-90a9-cc3ff4bbfc77.png)
